### PR TITLE
update CustomResourceDefinition version to v1

### PR DESCRIPTION
**Pull request by Thor-wl**
ref: https://github.com/volcano-sh/volcano/issues/1822
version `CustomResourceDefinition` will be updated to `v1` instead of `v1beta1` after k8s v1.22
Signed-off-by: Thor-wl <13164644535@163.com>

### CODE REVIEW

File: installer/helm/chart/volcano/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: A Helm chart for Volcano
 name: volcano
-version: 0.1
```
Comment (william-wang) [inline installer/helm/chart/volcano/Chart.yaml:4]: please change it to 1.5.

Comment (Thor-wl) [reply installer/helm/chart/volcano/Chart.yaml:4]: OK. Changed.

**Comment by k82cn** [comment]
Does that mean we will not support old k8s cluster?

**Comment by Thor-wl** [comment]
> Does that mean we will not support old k8s cluster?

Yes, the latest k8s version has already been v1.24. We cannot support so many old versions.

**Review by william-wang — APPROVED** [review]
/lgtm

**Comment by k82cn** [comment]
> > Does that mean we will not support old k8s cluster?
> 
> Yes, the latest k8s version has already been v1.24. We cannot support so many old versions.

We need to align with k8s release; for example, we should support at least 1.22, 1.23 & 1.24 :)

**Comment by k82cn** [comment]
please also update related document .

**Pull request merged**